tests/unit/test_logger.py

```diff
@@ -468,3 +468,25 @@ def test_logger_setlevel_resets_other_levels():
 
     logger.error("test")
     assert stream.getvalue() == "test\ntest\n"
+
+def test_logger_propagate_int():
+    stream1 = io.StringIO()
+    handler1 = picologging.StreamHandler(stream1)
+    logger1 = picologging.getLogger('A')
+    logger1.propagate = 0
+    logger1.addHandler(handler1)
+    logger1.setLevel(picologging.WARNING)
+
+    stream2 = io.StringIO()
+    handler2 = picologging.StreamHandler(stream2)
+    logger2 = picologging.getLogger('A.B')
+    logger2.propagate = 0
```

Comment:
This test currently passes even if propagate is set to 1 (or left to its default of True), due to our current lack of setting of parents. But in the future it should continue to pass once thats fixed.